merlin/study/script_adapter.py

```diff
@@ -35,13 +35,160 @@
 import logging
 
 from maestrowf.interfaces.script.localscriptadapter import LocalScriptAdapter
+from maestrowf.interfaces.script.slurmscriptadapter import SlurmScriptAdapter
 
 from merlin.common.abstracts.enums import ReturnCode
 
 
 LOG = logging.getLogger(__name__)
 
 
+class MerlinLSFScriptAdapter(SlurmScriptAdapter):
+    """
+    A SchedulerScriptAdapter class for slurm blocking parallel launches,
+    the LSFScriptAdapter uses non-blocking submits.
+    """
+
+    key = "merlin-lsf"
+
+    def __init__(self, **kwargs):
+        """
+        Initialize an instance of the MerinLSFScriptAdapter.
+        The MerlinLSFScriptAdapter is the adapter that is used for workflows that
+        will execute flux parallel jobs in a celery worker. The only configurable aspect to
+        this adapter is the shell that scripts are executed in.
+
+        :param **kwargs: A dictionary with default settings for the adapter.
+        """
+        super(MerlinLSFScriptAdapter, self).__init__(**kwargs)
+
+        self._cmd_flags = {
+            "cmd": "srun",
+            "ntasks": "-n",
+            "nodes": "-N",
+            "cores per task": "-c",
```

Comment:
What would you think about replacing `cores per task` with `cores_per_task`, for consistency? (`task_queue`, `max_retries`.) Same for `gpus per task`

Reply:
That is from maestro:
https://github.com/LLNL/maestrowf/blob/1bdcf987143e3eabe4235cc913ea910fe16cafca/maestrowf/interfaces/script/slurmscriptadapter.py#L92